release-notes/release-notes57.md

```diff
@@ -37,7 +37,12 @@ The following new features were added for the {{site.productname}} 5.7 release.
 
 The following enhancements were made for the {{site.productname}} 5.7 release.
 
-### Enhancement name
+### Improved behavior when using `lists` plugin on mixed content
+
+Improved behavior when using the `lists` plugin to add lists to selections containing:
+
+- Multiple text blocks.
+- Mixes of existing lists and text blocks.
```

Comment:
```suggestion
- Combinations of existing lists and text blocks.
```
`Mixes` just reads weird to me. Not entirely sure this is better though... @tylerkelly13